restaurant/urls.py

```diff
@@ -13,15 +13,19 @@
     1. Import the include() function: from django.conf.urls import url, include
     2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
 """
-from django.conf.urls import url
 from django.contrib import admin
 from django.contrib.auth import views as auth_views
-
+from django.conf.urls import url
+from views.index_view import index
+from views.login_view import login_view
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
-    url(r'^account/login/$', auth_views.login, name='login'),
-    url(r'^account/password_reset/$', auth_views.password_reset, name='password_reset'),
-    url(r'^account/password_reset/done/$', auth_views.password_reset_done, name='password_reset_done'),
+    url(r'^index/', index, name='index'),
+    url(r'^account/login/$', login_view, name='login'),
+    url(r'^account/password_reset/$', auth_views.password_reset,
+                                      name='password_reset'),
+    url(r'^account/password_reset/done/$', auth_views.password_reset_done,
+                                           name='password_reset_done'),
```

Comment:
In such case you probably shoud align this with an opening bracket.

Something like this:
```python
def function(one, two, three,
             four):
    pass
```